data-hub/src/hub-in-a-box/root/com.marklogic.hub/lib/flow-lib.xqy

```diff
@@ -274,7 +409,14 @@ declare function flow:run-flow(
     ()
 };
 
+(:~
+ : Construct an envelope
+ :
+ : @param $map - a map with all the stuff in it
+ : @return - the newly constructed envelope
+ :)
 declare function flow:make-envelope($map as map:map)
```

Comment:
Actually it's not a built in transform. that concept is so last week :)

we are not allowing the user to override the envelope.